src/components/Input/Input.stories.svelte

```diff
@@ -0,0 +1,24 @@
+<script>
+    import { Meta, Template, Story } from '@storybook/addon-svelte-csf'
+    import Input from './Input.svelte'
+</script>
+
+<Meta
+    title="Example/Input"
+    component={Input}
+    argTypes={{
+        label: { control: 'text' }
```

Comment:
Input component doesn't take a label. We should update the meta template to have the input properties it currently takes.